hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -600,6 +634,8 @@ public boolean doMetadataTableValidation() {
         LOG.warn("Metadata table validation failed ({}).", taskLabels);
         return false;
       }
+    } catch (HoodieValidationException e) {
+      throw e;
```

Comment:
Should we return false with a warn message?

Reply:
This is an error where HoodieValidationException is never thrown even if ignoreFailed is false. For example if validation exception is thrown by partition stat validation logic, it will be caught and will lead to validation being successful.